Azure/azure-cli
# [Storage] az storage account create/update: Enabled Secure File Transfer Protocol

**Description**

Waiting for https://github.com/Azure/azure-rest-api-specs/pull/11597 merged and SDK released.

**Testing Guide**

`az storage account create --enable-sftp`
`az storage account update --enable-sftp false`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
